tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/MySql/MySqlExerciser.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 New Relic, Inc. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#if !NET7_0 // there is no .NET 7 test for MySql
+#if !NET8_0 // there is no .NET 7 test for MySql
```

Comment:
I think this is breaking our MySql tests.

Reply:
Yeah, it was a leftover from my last round of test refactoring. We were building a .net 6, 7 and 8 version of the library but never using the .net 7 build. I should have removed the #if rather than updating it to net 8. Which I just did... ;-)